src/content/en/fundamentals/instant-and-offline/get-started/httprequests-5.md

```diff
@@ -0,0 +1,265 @@
+it more efficiently.
+
+## Combine Text Resources
+
+Many web pages use multiple stylesheets and script files. As we develop pages and add useful 
+formatting and behavior code to them, we often put pieces of the code into separate files for 
+clarity and ease of maintenance. Or we might keep our own stylesheet separate from the one the 
+Marketing Department requires us to use. Or we might put experimental rules or scripts in 
+separate files during testing. Those are all valid reasons to have multiple resource files.
+
+But because each file requires its own HTTP request, and each request takes time, we can speed 
+up page load by combining files; one request instead of three or four will certainly save 
+time. At first blush, this seems like a no-brainer -- just put all the CSS (for example) into 
+a master stylesheet and then remove all but one <link> from the page. Every extra file you 
+eliminate in this way removes one HTTP request and saves round-trip time. But there are caveats.
+
+For Cascading Style Sheets, beware the "C". Cascade precedence allows later rules to override 
+earlier ones without warning -- literally. CSS doesn't throw an error when a previously-defined 
+property is reset by a more recent rule, so just tossing stylesheets together is asking for 
+trouble. Instead, look for conflicting rules and determine whether one should always supercede 
+the other, or if one should use more specific selectors to be applied properly. For example, 
+consider these two simple rules, the first from a master stylesheet, the second imported from 
+a stylesheet provided by Marketing.
+
+```
+h2 { font-size: 1em; color: #000080; } /* master stylesheet */
+
+. . .
+
+h2 { font-size: 2em; color: #ff0000; } /* Marketing stylesheet */
+```
+
+Marketing wants their `h2`s to be prominent, while yours are meant to be more subdued. But 
+due to the cascading order, their rule takes precedence and every `h2` in the page will be 
+big and red. And clearly, you can't just flip the order of the stylesheets or you'll have 
+the same problem in reverse. 
+
+A little research might show that Marketing's `h2`s always appear inside a specific class 
+of `section`, so a tweak to the second rule's selector resolves the conflict. Marketing's 
+splashy `h2`s will still look exactly as they want, but without affecting `h2`s elsewhere 
+in the page.
+
+```
+h2 { font-size: 1em; color: #000080; }
+
+. . .
+
+section.product h2 { font-size: 2em; color: #ff0000; }
+```
+
+You may run into similar situations when combining JavaScript files. Completely different 
+functions might have the same names, or identically-named variables might have different 
+scopes and uses. These are not insurmountable obstacles if you actively look for them.
+
+Combining text resources to reduce HTTP requests is worth doing, but take care when 
+doing so. See **A Caveat** below.
+
+## Combine Graphical Resources
```

Comment:
This technique is often referred to as "spriting". It might be useful to use this term somewhere in this section to make it easier for folks that wish to research more about the technique later.